init/systemd/ipa.sysusers.base.conf

```diff
@@ -0,0 +1,5 @@
+u kdcproxy  -   "IPA KDC Proxy User"    -
+u dirsrv    -   -   -
```

Comment:
The only thing we achieve by declaring the user ourselves is supporting broken `389-ds-base` installs. I don't think we should do that and rather fail early if the user is missing.